README.rst

```diff
@@ -0,0 +1,39 @@
+edx Themes for LMS and Studio
+=============================
+This directory contains themes for LMS (Learning Management System) and Studio ,
+following directory structure should be followed for each theme
+
+
+.. code-block:: text
+
```

Comment:
should this example now include a "themes" directory, as though my-theme will be the theme for one of the sites in the instance?